server/src/index.js

```diff
@@ -47,17 +47,14 @@ async function start() {
   applyFileMiddelware(app);
   app.get('/health', (req, res) => res.sendStatus(200))
 
-  app.use('/graphql', altairExpress({
-    endpointURL: '/graphql'
-  }))
+  app.use('/graphql', altairExpress(config.altairConfig))
 
   // connect to db
   const client = await connect(config.db);
 
   const apolloConfig = {
     typeDefs: appTypeDefs,
     resolvers: appResolvers,
-    playground: config.playgroundConfig,
```

Comment:
If we run project without altairExpress above playground still appear. 
We missing required flags: https://www.apollographql.com/docs/apollo-server/features/graphql-playground.html#Enabling-GraphQL-Playground-in-production